### PR TITLE
Added Gold Ring of Swiftness

### PR DESCRIPTION
**Pull request by jeffsieu**

### CODE REVIEW

File: assets/prefabs/Equipment/Gold/GoldRingOfSwiftness.prefab [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "parent" : "engine:iconItem",
+    "Item" : {
+        "icon" : "Equipment:GoldRingOfSwiftness"
+    },
+    "DisplayName" : {
+        "name" : Gold Ring of Swiftness",
```
Comment (Cervator) [inline assets/prefabs/Equipment/Gold/GoldRingOfSwiftness.prefab]: You're missing a `"` here :-)

**Review by Cervator — CHANGES_REQUESTED** [review]
Nice ring! We need to better source the two images though, we need a page that shows the image origin and some sort of license that is compatible with us using them (like public domain, or CC BY 4.0)

Or you could just throw together a simple pixel art ring yourself and mark you as the author in the readme :)

**Comment by jeffsieu** [comment]
Alright I have created a new pixel art ring, similar to the previous one, this should solve any copyright issues!

**Comment by Cervator** [comment]
Yep! That works :-) 

Thanks!

**Comment by Cervator** [comment]
Merged via rebase in https://github.com/Terasology/Equipment/commit/f1adcd2d725b078706432bdc1240fe030232ada7 - closing as merged.

**Pull request closed without merging**